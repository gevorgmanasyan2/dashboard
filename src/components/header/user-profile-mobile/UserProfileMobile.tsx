import React, { FC } from 'react'
import { Menu, Dropdown } from 'antd'
import UserProfile from '../user-profile/UserProfile'
import { MenuOutlined } from '@ant-design/icons'
import './userProfileMobile.scss'

const UserProfileMobile: FC = () => {
  const menu = (
    <Menu className="userProfileMobile__container__dropdown">
      <Menu.Item key="0">
        <UserProfile />
      </Menu.Item>
    </Menu>
  )
  return (
    <div className="userProfileMobile__container">
      <Dropdown
        overlay={menu}
        trigger={['click']}
        placement="bottomCenter"
        overlayClassName="userProfileMobile__container__info"
      >
        <MenuOutlined />
      </Dropdown>
    </div>
  )
}

export default UserProfileMobile
