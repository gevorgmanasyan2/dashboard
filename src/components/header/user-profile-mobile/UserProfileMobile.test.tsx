/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import UserProfileMobile from './UserProfileMobile'

it('should render UserProfileMobile component', () => {
  expect(shallow(<UserProfileMobile />)).toMatchSnapshot()
})