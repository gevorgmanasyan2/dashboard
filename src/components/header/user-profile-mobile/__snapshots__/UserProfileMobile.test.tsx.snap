// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render UserProfileMobile component 1`] = `
<div
  className="userProfileMobile__container"
>
  <Dropdown
    mouseEnterDelay={0.15}
    mouseLeaveDelay={0.1}
    overlay={
      <Menu>
        <MenuItem>
          <UserProfile />
        </MenuItem>
      </Menu>
    }
    overlayClassName="userProfileMobile__container__info"
    placement="bottomCenter"
    trigger={
      Array [
        "click",
      ]
    }
  >
    <ForwardRef(MenuOutlined) />
  </Dropdown>
</div>
`;
